docroot/themes/humsci/humsci_basic/package.json

```diff
@@ -17,7 +17,7 @@
   "license": "MIT",
   "dependencies": {
     "bourbon": "^5.1.0",
-    "decanter": "^5.1.0"
+    "decanter": "^6.0.0-alpha.1"
```

Comment:
need to further analyze version 6 when it is released and what new things we might want to include from it (should be happening today)

Reply:
release notes here https://github.com/SU-SWS/decanter/releases/tag/6.0.0

Reply:
lots of general cleanup occurred, as well as adding two new components: secondary nav and card icon